server/statistics/statistics.go

```diff
@@ -0,0 +1,138 @@
+package statistics
+
+import (
+	"sync"
+	"time"
+
+        "github.com/ubports/ubuntu-push/logger"
+
```

Comment:
Please reformat this file according to the style of the other files in the project: use spaces instead of tabs, remove empty lines after opening braces and before closing braces, etc.

Reply:
Are you sure there are only spaces used? MC reports tabs here:
![grafik](https://user-images.githubusercontent.com/16186470/34198231-e652037c-e569-11e7-9a85-d72546b7ca35.png)

Reply:
Wow, you are right, this project is a mess :-(
Well, feel free to pick to coding standard you prefer then, but please make it consistent: use either tabs, or spaces (I strongly recommend using spaces :-) ).